testpar/t_filters_parallel.c

```diff
@@ -9956,6 +10008,18 @@ main(int argc, char **argv)
                                 continue;
                         }
 
+                        /*
+                         * If TestExpress is > 1, only run the multi-chunk I/O
+                         * configuration tests for the 'USE_SINGLE_DATASET' case,
+                         * as the 'USE_MULTIPLE_DATASETS' and 'USE_MULTIPLE_DATASETS_MIXED_FILTERED'
+                         * cases are more stressful on the file system.
+                         */
+                        if (test_express_level_g > 1) {
+                            if (((test_mode == USE_MULTIPLE_DATASETS) || (test_mode == USE_MULTIPLE_DATASETS_MIXED_FILTERED))
+                                    && (chunk_opt != H5FD_MPIO_CHUNK_ONE_IO))
```

Comment:
When the TestExpress level is > 1, skip testing of multi-chunk I/O in the multi-dataset I/O case since it tends to just hammer the filesystem and the multi_dset test already does a bit of testing of this case.